app/systemd/entry.sh

```diff
@@ -1,14 +1,65 @@
 #!/bin/bash
 
- set -m
+set -m
 
- # Send SIGTERM to child processes of PID 1.
+# This command only works in privileged container
+ip link add dummy0 type dummy &> /dev/null
+if [[ $? == 0 ]]; then
```

Comment:
I think this can be condensed as follows:

```suggestion
if ip link add dummy0 type dummy &> /dev/null; then
```

Reply:
@xginn8 good point, and yeah, I was mindlessly copy-pasting from the original PR https://github.com/balena-io-library/base-images/pull/573 and should have known better :P Will update here, and would you want to PR your comments and changes there? (or can do myself, and credit you, but I think you'd prefer the former.

Reply:
And also feed back on https://stackoverflow.com/a/32144661 that they could do better :P

Reply:
https://github.com/balena-io-library/base-images/pull/578 has been opened accordingly!